docs/developer-docs/defi/tokens/ledger/setup/icp_ledger_setup.mdx

```diff
@@ -111,7 +134,8 @@ Open the `dfx.json` file in your project's directory. Replace the existing conte
         "id": {
           "ic": "ryjl3-tyaaa-aaaaa-aaaba-cai"
         }
-      }
+      },
+      "init_arg" : "(variant { Init = record { minting_account = "MINTER_ACCOUNT_ID"; initial_values = vec { record { "DEFAULT_ACCOUNT_ID"; record { e8s = 10_000_000_000 : nat64; }; }; }; send_whitelist = vec {}; transfer_fee = opt record { e8s = 10_000 : nat64; }; token_symbol = opt \"LICP\"; token_name = opt \"Local ICP\"; } })"
```

Comment:
do we want to show all fields or just the ones we are setting?
```
(
  variant {
    Init = record {
      send_whitelist = vec {};
      token_symbol = opt "LICP";
      transfer_fee = opt record {
        e8s = 10_000 : nat64;
      };
      minting_account = "MINTER_ACCOUNT_ID";
      maximum_number_of_accounts = null;
      accounts_overflow_trim_quantity = null;
      transaction_window = null;
      max_message_size_bytes = null;
      icrc1_minting_account = null;
      archive_options = null;
      initial_values = vec {
        record {
          "DEFAULT_ACCOUNT_ID";
          record {
            e8s = 10_000_000_000 : nat64;
          }
        }
      };
      token_name = opt "Local ICP";
      feature_flags = null;
    };
  }
)
```

Reply:
IMO just the ones we are setting